lib/tasks/licenses.rake

```diff
@@ -14,19 +16,18 @@ namespace :licenses do  # rubocop:disable Metrics/BlockLength
 
   desc 'Check license compliance of dependencies'
   task :compliance do
-    warn 'Checking license action items...'
-    unless cli.action_items
-      warn '*** License compliance test failed  ***'
-      exit 1
-    end
+    require 'license_finder'
+
+    warn 'Checking action items...'
+    exit(1) unless LicenseFinder::CLI::Main.new.action_items
 
     warn 'Checking license report is up to date...'
-    Tempfile.new(report_path.to_s) do |temp_file|
+    Tempfile.create do |temp_file|
```

Comment:
shouldn't this be `Tempfile.open`?

Reply:
I hope not!

Reply:
I guess I copy pasted it from somwhere, I can't say there's a big difference:
>create
>Creates a temporary file as usual File object [...]
>If a block is given, then a File object will be constructed, and the block is invoked with the object as the argument. >The File object will be automatically closed and the temporary file is removed after the block terminates. The call >returns the value of the block.

>open
>Creates a new Tempfile [...]
>If a block is given, then a [Tempfile](https://ruby-doc.org/stdlib-2.5.3/libdoc/tempfile/rdoc/Tempfile.html) object will >be constructed, and the block is run with said object as argument. The [Tempfile](https://ruby-doc.org/stdlib->2.5.3/libdoc/tempfile/rdoc/Tempfile.html) object will be automatically closed after the block terminates. The call >returns the value of the block.

Reply:
hmm, with a block it seems `#create` is the recommended way. `#open` should also work though. Cool.